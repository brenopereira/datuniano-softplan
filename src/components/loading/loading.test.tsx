import React from 'react';
import renderer from 'react-test-renderer';

import { Loading } from '.';

test('Validate render Loading component', () => {
    const component = renderer.create(<Loading />);

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});
