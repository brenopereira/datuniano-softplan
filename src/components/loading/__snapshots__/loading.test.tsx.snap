// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validate render Loading component 1`] = `
<div
  className="sc-bdnxRM eCsFZR"
>
  Aguarde, carregando os dados!
</div>
`;
